<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabcoin">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>600</width>
      <height>345</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="movable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="tabeth">
     <attribute name="title">
      <string>ETH</string>
     </attribute>
     <widget class="QPushButton" name="btn_eth_submit">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>160</y>
        <width>75</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>submit</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_eth_open">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>10</y>
        <width>60</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>open</string>
      </property>
     </widget>
     <widget class="QLabel" name="lab_eth_to">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>50</y>
        <width>50</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>to:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lab_eth_from">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>15</y>
        <width>50</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>from:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_eth_from">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>15</y>
        <width>450</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lab_eth_amount">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>120</y>
        <width>50</width>
        <height>20</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>amount:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_eth_amount">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>120</y>
        <width>180</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_eth_to">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>50</y>
        <width>450</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lab_eth_fee">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>120</y>
        <width>50</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>fee:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_eth_fee">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>120</y>
        <width>180</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lab_eth_token">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>85</y>
        <width>50</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>token:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_token">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>85</y>
        <width>130</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_eth_insert">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>80</y>
        <width>70</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>add</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>120</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>ETH</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabbtc">
     <attribute name="title">
      <string>BTC</string>
     </attribute>
     <widget class="QPushButton" name="btn_btc_open">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>10</y>
        <width>60</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>open</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_btc_to">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>50</y>
        <width>450</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_branch">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>85</y>
        <width>130</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_btc_insert">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>80</y>
        <width>70</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>add</string>
      </property>
     </widget>
     <widget class="QLabel" name="lab_btc_fee">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>120</y>
        <width>50</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>fee:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lab_btc_to">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>50</y>
        <width>50</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>to:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_btc_from">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>15</y>
        <width>450</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_btc_amount">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>120</y>
        <width>180</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_btc_submit">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>160</y>
        <width>75</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>submit</string>
      </property>
     </widget>
     <widget class="QLabel" name="lab_btc_from">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>15</y>
        <width>50</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>from:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_btc_fee">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>120</y>
        <width>180</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lab_btc_amount">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>120</y>
        <width>50</width>
        <height>20</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>amount:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lab_btc_branch">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>85</y>
        <width>50</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>branch:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>120</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>BTC /kVB</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menufile">
    <property name="title">
     <string>file</string>
    </property>
    <addaction name="actionopen"/>
   </widget>
   <widget class="QMenu" name="menuview">
    <property name="title">
     <string>view</string>
    </property>
    <addaction name="actiontx"/>
    <addaction name="actionbalance"/>
   </widget>
   <widget class="QMenu" name="menuconfig">
    <property name="title">
     <string>config</string>
    </property>
    <addaction name="actiongenerate"/>
   </widget>
   <addaction name="menufile"/>
   <addaction name="menuview"/>
   <addaction name="menuconfig"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionopen">
   <property name="text">
    <string>open</string>
   </property>
  </action>
  <action name="actiontx">
   <property name="text">
    <string>tx</string>
   </property>
  </action>
  <action name="actiongenerate">
   <property name="text">
    <string>generate</string>
   </property>
  </action>
  <action name="actionbalance">
   <property name="text">
    <string>balance</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
